### PR TITLE
Added .env and POAP instructions to readme

### PR DESCRIPTION
**Pull request by wackerow**
## Description
- Adds some more descriptions to stack and examples for our conventions
- Adds step for setting up GitHub API Token in `.env` to local environment setup
- Introduces POAP section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -318,9 +385,30 @@ export const query = graphql`
...
+<hr style="margin-top: 3em; margin-bottom: 3em;">
 
-[![Discord](https://img.shields.io/discord/714888181740339261?color=1C1CE1&label=ethereum.org%20%7C%20Discord%20%F0%9F%91%8B%20&style=flat-square)](https://discord.gg/CetY6Y4)
+![POAP Logo](src/assets/poap-logo.svg)
+
+## Claim your POAP!
+
+### What is POAP?
+
+> The Proof of Attendance Protocol is a dapp that distributes badges in the form of ERC-721 tokens to prove you participated in an event. More info on the dapp and the multiclient token can be found [here](https://www.poap.xyz/).
```
Comment (ryancreatescopy) [inline README.md:396]: > The Proof of Attendance Protocol is a dapp that distributes badges in the form of ERC-721 tokens to prove you participated in an event. [More on POAPs](https://www.poap.xyz/).

File: README.md [hunk 2]
```diff
@@ -318,9 +385,30 @@ export const query = graphql`
...
+
+### ethereum.org 2020 Contributor POAP
+
+- If you commited any changes in 2020 that were merged to our repo, you have a POAP waiting!
+- This includes our dedicated supporters on Crowdin who contributed to our translation efforts
```
Comment (ryancreatescopy) [inline README.md:401]: "merged into"

"This includes our dedicated translators on Crowdin."

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -318,9 +385,30 @@ export const query = graphql`
...
+> The Proof of Attendance Protocol is a dapp that distributes badges in the form of ERC-721 tokens to prove you participated in an event. [More on POAPs](https://www.poap.xyz/).
+
+### ethereum.org 2020 Contributor POAP
+
+- If you commited any changes in 2020 that were merged into our repo, you have a POAP waiting!
+- This includes our dedicated translators on Crowdin
+
+  [![Discord](https://img.shields.io/discord/714888181740339261?color=1C1CE1&label=Claim%20Your%20POAP!%20%7C%20Discord%20%F0%9F%91%8B%20&style=flat)](https://discord.gg/CetY6Y4)
+
+- 👆 To claim your Contributor POAP, join our Discord server and paste a link to your contribution in the #POAP channel
```
Comment (samajammin) [inline README.md:405]:
```suggestion
- 👆 To claim your Contributor POAP, join our Discord server and paste a link to your contribution in the #contributing channel
```

Comment (samajammin) [reply README.md:405]: Let's just use the existing #contributing channel

**Review by samajammin — APPROVED** [review]
Looks great! Made 1 minor change.

**Pull request merged**